src/table.ts

```diff
@@ -18,14 +18,43 @@
 
 import {promisifyAll} from '@google-cloud/promisify';
 import * as is from 'is';
-import {TransactionRequest} from './transaction-request';
+import {ServiceError} from 'grpc';
+import * as through from 'through2';
+
+import {ReadRequest, Json} from './codec';
+import {Database} from './database';
+import {DatabaseAdminClient as d, SpannerClient as s} from './v1';
+import {PartialResultStream, Row} from './partial-result-stream';
+import {TransactionOptions} from './transaction';
+
+type Key = string|string[];
```

Comment:
I can do that, I was holding off because I think these definitions will actually move to different files (Transaction/Database) eventually. I've been using the typings as an excuse to refactor certain parts of the client and one of my goals (that I've been wanting to do for a while) is kill of `TransactionRequest`. Doing `Table` first was just going to make life easier when doing `Transaction`.